python/ray/dataframe/dataframe.py

```diff
@@ -698,24 +699,127 @@ def cov(self, min_periods=None):
             "github.com/ray-project/ray.")
 
     def cummax(self, axis=None, skipna=True, *args, **kwargs):
-        raise NotImplementedError(
-            "To contribute to Pandas on Ray, please visit "
-            "github.com/ray-project/ray.")
+        """Perform a cumulative maximum across the DataFrame.
+
+        Args:
+            axis (int): The axis to take maximum on.
+            skipna (bool): True to skip NA values, false otherwise.
+
+        Returns:
+            The cumulative maximum of the DataFrame.
+        """
+        if(axis == 1):
```

Comment:
There are a couple instances if this that are probably linting errors. Should just be `if axis == 1:`

Reply:
Ok, I will fix these.